metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -542,6 +542,39 @@ object Messages {
     }
   }
 
+  object UpdateScalafmtConf {
+
+    def letUpdate = new MessageActionItem("Let Metals update .scalafmt.conf")
+    def createMessage(dialect: ScalafmtDialect): String = {
+      s"Some source can't be formatted by scalafmt because they requires `runner.dialect = ${dialect.value}` setting."
```

Comment:
```suggestion
    def createMessage(dialect: ScalafmtDialect): String = {
      s"Some sources can't be formatted by scalafmt because they require `runner.dialect = ${dialect.value}` setting."
```

Reply:
As always, thx for grammar check!
I reworked this message a little bit could you check it one more time?